docs/pages/ds-audio_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,70 @@
+# Audio Settings L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [Overview](#overview)
+- [References](#references)
+- [Audio Streams Requirement](#audio-streams-requirement)
+- [Level 3 Test Procedure](#level-3-test-cases-high-level-overview)
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL`    - Hardware Abstraction Layer
+- `API`    - Application Programming Interface
+- `L2`     - Level 2 Testing
+- `L3`     - Level 3 Testing
+- `DS`     - Device Settings
+- `ARC`    - Audio Return Channel
+- `HDMI`   - High-Definition Multimedia Interface
+- `LE`     - Loudness Equivalence
+- `DRC`    - Dynamic Range Control
+- `MI`     - Media Intelligent
+- `MS12`   - MultiStream 12
+- `MS11`   - MultiStream 11
+- `DAP`    - Digital Audio Processing
+- `PCM`    - Pulse Code Modulation
+- `WAV`    - Waveform
+- `DUT`    - Device Under Test
+- `NA`     - Not Applicable
+- `RAFT`   - Rapid Automation Framework for Testing
+
+## Overview
+
+This document describes the L3 Test Procedure for the Device Settings Audio module.
+
+## References
+
+- dsAudio HAL Interface - [dsAudio.h](https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsAudio.h)
+- High Level Test Specification - [ds-audio_High-Level_TestSpecification.md](https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/docs/pages/ds-audio_High-Level_TestSpecification.md)
```

Comment:
this is too wide as a url needs to be a specific version, but can't you include a relative rather than ah HTTP link? So it's always aligned to the code your commiting and the tag your using.

Reply:
done